### ISSUE TITLE
"UnhandledPromiseRejectionWarning: SerializationException" in DynamoDB

### ISSUE THREAD

**Issue by trivikr** [aws/aws-sdk-js-v3#870]
**Describe the bug**
"UnhandledPromiseRejectionWarning: SerializationException" in DynamoDB

**SDK version number**
```console
$ yarn list @aws-sdk/client-dynamodb
└─ @aws-sdk/client-dynamodb@1.0.0-alpha.16
```

**Is the issue in the browser/Node.js?**
Both

**To Reproduce (observed behavior)**
Can be reproduced in repro repo https://github.com/aws-samples/aws-sdk-js-tests

Code:
```javascript
const AWS = require("aws-sdk");
const { DynamoDB } = require("@aws-sdk/client-dynamodb");
const { REGION } = require("./config");

(async () => {
  let response;

  const v2Client = new AWS.DynamoDB({ region: REGION });
  response = await v2Client.listTables().promise();
  console.log("Data returned by v2:");
  console.log(JSON.stringify(response, null, 2));

  const v3Client = new DynamoDB({ region: REGION });
  response = await v3Client.listTables({});
  console.log("\nData returned by v3:");
  console.log(JSON.stringify(response, null, 2));
})();
```

Output:
```console
$ node src/node.js 
Data returned by v2:
{
  "TableNames": [
    "CUSTOMER_LIST",
    "MysfitsTable",
    "aws-js-sdk-workshop-notes",
    "aws-sdk-js-integration-test",
    "notes",
    "notes-claudia",
    "release-status"
  ]
}
(node:12849) UnhandledPromiseRejectionWarning: SerializationException: SerializationException
    at deserializeAws_json1_0ListTablesCommandError (/Users/trivikr/workspace/aws-sdk-js-tests/node_modules/@aws-sdk/client-dynamodb/dist/cjs/protocols/Aws_json1_0.js:1367:41)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(node:12849) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 4)
(node:12849) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Expected behavior**
v3 returns list of tables

**Comment by kstich** [comment]
Request body for this is undefined instead of an empty `{}`, related to #860 + awslabs/smithy-typescript#118. Verified working with `{}` as the body.

**Comment by srchase** [comment]
Confirmed that `listTables` works in `@aws-sdk/client-dynamodb@1.0.0-alpha.18`

**Comment by trivikr** [comment]
Verified that issue is fixed in `@aws-sdk/client-dynamodb@1.0.0-alpha.18` in master branch of https://github.com/aws-samples/aws-sdk-js-tests
